portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,60 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+html{
+    height: 100%;
+    width: 100%;
 }
+ 
+body{
+    height: 100%;
+    margin: 0;
+    background: linear-gradient(45deg,
+      rgba(20, 18, 19, 0.9) 30%,
+      rgba(0, 94, 105, 0.9) 100%),
+    url(Images/github_pattern.svg);
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+}
+ 
+#content {
```

Comment:
nit: Maybe rename this to content-container?

Reply:
I erased because it was duplicated!